hebiros_gazebo_plugin/launch/boilerplate_motor_plugin_test_01.launch

```diff
@@ -0,0 +1,11 @@
+<launch>
```

Comment:
Do we need these files?  It seems like we should delete these launch files before merging back in